xray/src/build_quadtree.rs

```diff
@@ -135,12 +135,12 @@ pub fn run<T: Extension>(data_provider_factory: DataProviderFactory) {
         .unwrap()
         .parse::<usize>()
         .expect("num_threads could not be parsed.");
-    let tile_size = args
+    let tile_size_px = args
         .value_of("tile_size")
         .unwrap()
         .parse::<u32>()
         .expect("tile_size could not be parsed.");
```

Comment:
nit: tile_size -> tile_size_px

Reply:
See above

Reply:
I don't see anything above, but I think I know what you mean. I misread the code - I thought you changed the names of some arguments,.